stable/xray/values.yaml

```diff
@@ -55,6 +55,14 @@ xray:
         type: {{ .Values.database.type }}
         driver: {{ .Values.database.driver }}
       {{- end }}
+      {{- if and (not .Values.rabbitmq.enabled) .Values.rabbitmq.external }}
```

Comment:
This if is wrong. 
If .Values.rabbitmq.external will always return true because it is a map with underlying keys in the values.yaml file

Reply:
Fixed

Reply:
I don't understand the relation between `enableConnectionFromEnvironment` and the external RabbitMQ